src/js/Router.js

```diff
@@ -15,6 +16,12 @@ import WebviewSelection from './screens/WebviewSelection';
 import WebviewScreen from './screens/Webview';
 import DrawerLinks from './components/DrawerLinks';
 
+export function handleQuickActionsNavigation (navigation){
+	DeviceEventEmitter.addListener('quickActionShortcut', data => {
```

Comment:
You can omit the `=> {` as you are returning the `navigation.replace` 🙂

Reply:
That's true, tnx